wherehows-web/app/styles/components/nacho/_nacho-select.scss

```diff
@@ -0,0 +1,51 @@
+@import "../../abstracts/variables";
+@import "../../abstracts/mixins";
+
+/**
+ * <select> primary color, borders, icons, etcetera...
+ */
+$color: set-color(grey, light);
+
+.nacho-select {
+  display: flex;
+  align-items: center;
+  height: 34px;
+  width: 300px;
+  border: 1px solid shade($color, 20%);
+  background-color: set-color(white, base);
+  padding: 0 14px;
+
+  select {
+    @include prefix(appearance, none, ('webkit', 'moz'));
+
+    &::-ms-expand {
```

Comment:
::-ms-expand ? It achieves the same effect as appearance: none, or -webkit-appearance: none etc. MS browsers don't have the same prefix format for this rule so I can't use my prefix method. At least not cleanly